### ISSUE TITLE
Formatting Overridden by Prettier

### ISSUE THREAD

**Issue by entozoon** [panoply/vscode-liquid#34]
If you use Prettier, its HTML  formatting trumps this plugin and makes a huge hot mess.
There is a workaround which is that you can set your workspace settings along the lines of
```json
{  
  "settings": {
    "editor.formatOnSave": true,
    "[html]": {
      "editor.defaultFormatter": "sissel.shopify-liquid"
    }
  }
}
```
Which will force this plugin to be the default formatter for HTML language files.
Obviously that'll apply to filetypes other than .liquid, which isn't ideal but most likely fine for a Jekyll/Shopify project.
Perhaps it could be built-in as an option, to this plugin? Might not be possible. Either way this issue could be closed and stay here to help anyone with the same problem.

PS: Fantastic plugin, thank you @panoply ! The "Generate a .liquidrc file" feature is genius.

**Comment by panoply** [comment]
Can you create another issue for this? @entozoon

**Comment by datapolitical** [comment]
Is this still a workable solution? I can't figure out how to install the correct plugin to allow prettier to process my html files with liquid.

**Comment by panoply** [comment]
🚢  Shipped [v3.0.0](https://github.com/panoply/vscode-liquid/releases/tag/v3.0.0)

Take a look at the readme. You need to set the `editor.defaultFormatter`
